### ISSUE TITLE
'Prerequisites' section - grammatical issues

### ISSUE THREAD

**Issue by harizhisham** [mschermann/data_viz_reader#1378]
https://github.com/mschermann/data_viz_reader/blob/670e2582620adbc83086a5b018abdeae1293685d/06-process.Rmd#L37-L51

In ref to the above section of the prerequisites of a data viz project, I propose that we collectively reword all of these sentences to be more detailed, grammatically correct, and provide examples of projects that might be suitable for each level of coding skill(s).

**Comment by srivasud** [comment]
@harizhisham  agreed!

**Comment by AjayDeshmukh14** [comment]
I also agree with you @harizhisham, this gives a better understanding of the topic.
